api/python/notebooks/experimental/mean_variance.ipynb

```diff
@@ -0,0 +1,726 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c4b089f9-5304-4479-b739-20792e12595a",
+   "metadata": {},
+   "source": [
+    "# Out-of-core mean and variance calculation\n",
+    "\n",
+    "This tutorial describes use of the cellxgene_census.experimental.pp API for calculating out-of-core mean and variance in the Census. The variance calculation is performed using [Welford's online algorithm](https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance).\n",
+    "\n",
+    "There is a single API available: \n",
+    "\n",
+    "`mean_variance()`: calculates the mean and the variance for an `ExperimentAxisQuery`. The following additional arguments are supported:\n",
+    "- `layer`: the X layer used for the calculation, defaults to `raw`\n",
+    "- `axis`: the axis along which the calculation is performed. Specify 0 for the `var` axis and 1 for the `obs` axis\n",
+    "- `calculate_mean`: if False, do not include the mean in the result\n",
+    "- `calculate_variance`: if False, do not compute the variance.\n",
+    "- `ddof`: _Delta Degrees of Freedom_: the divisor used in the calculation for variance is N - ddof, where N represents the number of elements.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "a06549db-60c5-49b6-b57d-29ee6db2c61c",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Import packages\n",
+    "import cellxgene_census\n",
+    "from cellxgene_census.experimental.pp import mean_variance"
```

Comment:
`import somacore a soma`

Reply:
You probably want `import tiledbsoma as soma`.  You almost never want to import somacore directly.

Reply:
oh yeah, that was my bad typing ... I had the right version on my own testing code (facepalm)